rpmlb/downloader/base.py

```diff
@@ -58,3 +68,25 @@ def after(self, work, **kwargs):
 
     def download(self, package_dict, **kwargs):
         raise NotImplementedError('Implement this method.')
+
+    def _is_download_skipped(self, package_dict: dict, **kwargs):
+        """Return if skip a download for a pacakge.
+
+        Override if needed.
+
+        Keyword arguments:
+            package_dict: A dictionary of package metadata.
+            kwargs: option arguments.
+        """
+
+        if not package_dict:
+            raise ValueError('package_dict is required.')
+
+        is_skipped = False
+
+        dist = kwargs.get('dist')
+        if dist and 'dist' in package_dict:
+            pattern = re.compile(package_dict['dist'])
+            if not pattern.match(dist):
```

Comment:
@khardix Thanks for mentioning it. I did not know the formula `re.match(pattern, string)`. So, I fixed with it.